### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by jaehyi**

### CODE REVIEW

**Comment by mister-dl** [comment]
I approve

File: CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # Code owners for this project are the following:
-*       @dakinwerneburg @dmitrylandy @phillip-m-tham
+*       @dakinwerneburg @dmitrylandy @jaehyi @phillip-m-tham
```
Comment (dakinwerneburg) [inline CODEOWNERS:2]: Looks good.  Need others?

Comment (mister-dl) [inline CODEOWNERS:2]: This is a fine addition to the team

**Review by dakinwerneburg — APPROVED** [review]

**Review by mister-dl — CHANGES_REQUESTED** [review]
I dont like you

**Review by jaehyi — COMMENTED** [review]
Change made

**Review by mister-dl — APPROVED** [review]
I do actually like you

**Pull request merged**